doc/navigating.rst

```diff
@@ -65,18 +71,46 @@ Using the Account Settings Window
 .. index:: account settings, user, password, Server URL
 
 Click **Settings** in the right-click menu to see a summary of your ownCloud 
-account settings. This shows which ownCloud account you are connected to (or accounts, if you have more than one) your 
+account settings, or left-click your systray icon. This shows which ownCloud 
+account you are connected to (or accounts, if you have more than one) your 
 quota status, and a window for managing your synchronization settings.
 
 .. image:: images/client6.png
    :alt: Account settings window
 
-At the top of the window are tabs for each configured sync account, and three others for Activity, General and Network settings.
+At the top of the window are tabs for each configured sync account, and three 
+others for Activity, General and Network settings. On your account tabs you 
+have the following features:
+
+* Connection status, showing which ownCloud server you are connected to, and 
+  your ownCloud username.
+* A **Remove Account** button, which deletes your account but does not delete 
+  your data files.
+* Used and available space on the server.
+* Current synchronization status.
+* **Add Folder Sync Connection** button, which is active only when you have 
```

Comment:
Havn't used the new sync client 2.0 yet but is this correct? I know from previous versions that this botton is only greyed out if the root folder of the remote oC is synced.